### ISSUE TITLE
Redirect Request: Education Migration Part 2

### ISSUE THREAD

**Issue by jennymayoco** [department-of-veterans-affairs/va.gov-team#85453]
### Status
**[2024-07-08] [Fran]** Unblocked. Targeting Sprint 8.
**[2024-06-06] [Fran]** Moved to Next refinement, although it's pending review and approval from stakeholders.

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track different steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

| POC | Github alias(es) |
--- | --- 
| OCTO point of contact: | @bethpotts  |
| Content point of contact: | @d-beyer @aprocik1  |
| IA point of contact: | @jennymayoco |

### Type of request
- [X] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you need Public Websites to implement this redirect?
- [X] Yes
- [ ] No - Please note: Redirect still needs approval from IA and PW will do PR review

### Implementation date
When does this request need to be live? **NOTE:** Redirects deploy M-Th. Friday launches should be avoided.
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [X] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL | Status
---  |  --- | ---
~~https://benefits.va.gov/gibill/isaksonroe/ev_faqs.asp **and all casing**~~ | ~~https://www.va.gov/resources/enrollment-verification-for-post-911-gi-bill-faqs/~~ | ~~Pending stakeholder review.~~ Confirming destination link with Sitewide Content. Update from Derek: "The plan for these FAQs has changed / been put off. You can remove this from this batch of redirects."
~~https://benefits.va.gov/gibill/isaksonroe/verification_of_enrollment.asp **and all casing**~~ | ~~https://www.va.gov/education/verify-school-enrollment/~~ | ~~Pending stakeholder review~~ holding off on this one for this batch until new FAQ page is live
https://www.benefits.va.gov/GIBILL/FGIB/Reallocation.asp **and all casing** | https://www.va.gov/education/survivor-dependent-benefits/transferred-benefits/ | Approved - no additional stakeholder feedback
https://www.benefits.va.gov/gibill/regional_processing.asp **and all casing** | https://www.va.gov/resources/regional-processing-office-addresses-for-gi-bill-applications/ | Approved - no additional stakeholder feedback
https://www.benefits.va.gov/gibill/resources/education_resources/debt_info.asp **and all casing** | https://www.va.gov/resources/how-your-reason-for-withdrawing-from-a-class-affects-your-va-debt/ | Approved - no additional stakeholder feedback

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [X] Requesting team: Above information is provided
- [x] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jennymayoco** [comment]
Hi @jilladams @FranECross - this redirect ticket is part 2 of Education migration work (part 1 #82806). Similar to the first ticket, our next step is to communicate the listed redirects to stakeholders for their review and approval, but wanted to put this on your radar for your upcoming planning as well. We'll follow up with any updates on when these are good to move forward. Thanks!

**Comment by jennymayoco** [comment]
Update - email comms has been sent out to stakeholders today

**Comment by jennymayoco** [comment]
Hi @FranECross @jilladams - these are good to go! I crossed out the first two redirects and added notes. We'll be holding on those for this batch. thanks!

**Comment by jennymayoco** [comment]
hi @jilladams @FranECross! checking in on this redirect to see if there are any updates on implementation timeline

**Comment by FranECross** [comment]
Thanks for checking, @jennymayoco ! I'm planning out the next sprint, which starts on 7/10, and this is slotted for that sprint (probably early in the sprint).

**Comment by randimays** [comment]
The redirects were just merged. I believe we have missed the deploy cutoff for today and it will merge into production tomorrow morning.

**Comment by jennymayoco** [comment]
@randimays sounds good, thank you for the update!

**Comment by jennymayoco** [comment]
Hi @randimays I'm still seeing the old URLs. Since there's no deploy on Fridays, will these be pushed out on Monday? Just want to confirm when I should be checking that all looks good in prod

**Comment by randimays** [comment]
@jennymayoco Sorry for the confusion! These redirects are handled differently than the normal, so they're changed in a different codebase which deploys M-F at 12pm ET.

If the redirects are `va.gov/url` -> `va.gov/different-url`, you can expect a M-Th deploy around 10am ET.

If the redirects are `benefits.va.gov/url` -> `va.gov/different-url`, or any other subdomain (like `cem.va.gov`) to `va.gov`, you can expect a M-F deploy around 12pm ET.

The deploy should happen in an hour, provided there are no impediments to release. I'll update here when it's ready. Thank you!

**Comment by randimays** [comment]
@jennymayoco These redirects have been deployed to production and look good to me. Could you please validate on your end, and close the ticket if all looks good?

**Comment by jennymayoco** [comment]
Confirming the redirects are going to the correct destination. Thank you for your support with these and for the background on deployment times @randimays !!

**Comment by FranECross** [comment]
Closing per Jenny's confirmation above.
